runironic-inspector.sh

```diff
@@ -66,12 +69,19 @@ if [ -n "${HTTP_BASIC_HTPASSWD}" ]; then
 fi
 
 # Configure auth for ironic client
-CONFIG_OPTIONS="--config-file /etc/ironic-inspector/inspector-dist.conf --config-file ${CONFIG}"
+CONFIG_OPTIONS="--config-file ${CONFIG}"
 auth_config_file="/auth/ironic/auth-config"
 if [ -f ${auth_config_file} ]; then
     CONFIG_OPTIONS+=" --config-file ${auth_config_file}"
 fi
 
 ironic-inspector-dbsync --config-file /etc/ironic-inspector/ironic-inspector.conf upgrade
 
+if [[ $INSPECTOR_REVERSE_PROXY_SETUP == "true" ]]
+then
+  build_j2_config $APACHE_CONFIG > $APACHE_CONFIG
+  sed -i "/Listen 80/c\#Listen 80" /etc/httpd/conf/httpd.conf
+  /usr/sbin/httpd 
```

Comment:
Not sure I'm fond of having two processes in one container. Maybe we need to split the httpd entry point as is done in ironic?